content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,75 @@
+
+## Traffic Management
+
+- **Added** the capability for Istio to automatically determine protocol types avoiding the need for specifying the mandatory container ports field in the pod spec.
+- **Added** a mode to the Gateway API for mutual TLS operation that is capable of reading certificates via SDS.
+- **Added** enhancements to the `EnvoyFiltier` API such that all configuration can be added or updated.
+- **Fixed** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues with protocols like MySQL and MongoDB where the service communicates over the network first in permissive mutual TLS mode.
+- **Improved** Redis load balancer defaults to `MAGLEV` when using the Redis proxy.
+- **Improved** traffic management to direct traffic to the same region and zone.
+- **Improved** Pilot performance.  CPU utilization may be up to 10 times lower.
+- **Improved** the capabilities of `ServiceEntries` to enable the same hostname in different namespaces.
+- **Improved** the capability of the Sidecar API to customize `OutboundTrafficPolicy`.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
```

Comment:
Think we should mention the newly added `/ctrlz` and `/debug` endpoints in these notes

Reply:
sorry for the dumb q, is citadel agent used only when SDS is enabled?  If so, I suggest we organize SDS related together since it is not enabled by default.

Reply:
Citadel agent is used 1) when SDS is enabled 2) when secure ingress is used (for this, the agent is embedded in ingress pod, and cluster isn't necessarily SDS enabled). Agreed we should group all SDS related updates together at the bottom of the section

Reply:
@myidpt @Monkeyanator @linsun I believe your comments have been addressed in followup pushes.  If you agree, please click `Resolve conversation`.  If not, please state your case :)

Reply:
I'm ok with not grouping the SDS related things together, since many of the items in security are SDS related.
But I think it is more precise to make this item an "improvement" rather than a "addition". So I moved this to line 31. Please see if that makes sense to you @sdake .

Reply:
accepted.